inbox/crispin.py

```diff
@@ -881,12 +882,8 @@ def all_uids(self) -> List[int]:
                 raise
 
         elapsed = time.time() - t
-        log.debug(
-            "Requested all UIDs", search_time=elapsed, total_uids=len(fetch_result)
-        )
```

Comment:
fetch_result is now a generator, if we wanted to know the length here we would need to convert it to list - which would use much more memory. Anyway it's not really interesting to know the size here.